pages/[id].js

```diff
@@ -62,7 +57,7 @@ const Styledarticle = styled.p`
   padding-right: 1.5rem;
 `;
 
-export default function ActivityCardDetails() {
+export default function ActivityCardDetails({ activities }) {
```

Comment:
Just a suggestion: Your code might look "cleaner" if you declare components inside separate files and import them in the pages. Also, when declaring the function of a page, you could give it a naming with "Page" in the end.